.github/actions/publish/action.yml

```diff
@@ -44,6 +44,12 @@ runs:
         node-version: ${{ inputs.NODE_VERSION }}
         cache: npm
 
+    - name: Set up Python 3.11
```

Comment:
This is needed because of an issue with python 3.12 and node-gyp https://github.com/nodejs/node-gyp/issues/2869

I am still doing the real fix so we are able to drop it bubbling up PRs bumping node-gyp https://github.com/balena-io-modules/drivelist/pull/453

But because we still have a dependency on etcher-image-writer (which requires a major with breaking changes to remove) we still have old node-gyp stuff around, but being worked on